modules/ROOT/pages/xamarin/using-metrics-sdk.adoc

```diff
@@ -0,0 +1,24 @@
+= AeroGear Services Metrics SDK
+
+This document explains how to use the AeroGear Metrics SDK on the Xamarin platform and integrate it with an App.
+
+== Provided metrics
+
+The default metrics contain mobile app and mobile device metrics.
```

Comment:
This isn't clear - does the he default metrics **service**/**sdk**/**something else** contain a mobile and a mobile device metrics?